projects/packages/videopress/src/class-initializer.php

```diff
@@ -302,8 +302,9 @@ public static function render_videopress_video_block( $block_attributes, $conten
 			$videopress_url = wp_kses_post( $videopress_url );
 			$oembed_html    = apply_filters( 'video_embed_html', $wp_embed->shortcode( array(), $videopress_url ) );
 			$video_wrapper  = sprintf(
-				'<div class="%s">%s %s</div>',
+				'<div class="%s" id="jetpack-videopress-%s">%s %s</div>',
```

Comment:
Nit: It would be good to update this to 
```suggestion
				'<div class="%1$s" id="jetpack-videopress-%2$s">%3$s %4$s</div>',
```